x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/inference/action/DeleteInferenceEndpointAction.java

```diff
@@ -142,6 +164,11 @@ public String toString() {
             for (String entry : pipelineIds) {
                 returnable.append(entry).append(", ");
             }
+            returnable.append(", semanticTextFieldsByIndex: ");
```

Comment:
You can use `org.elasticsearch.common.Strings::toString(ToXContent toXContent)` for a simple toString implementation - `return Strings.toString(this);`